packages/office-ui-fabric-react/src/utilities/selectableOption/SelectableDroppableText.Props.ts

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { IRenderFunction } from '../../Utilities';
+import { ICalloutProps } from '../../Callout';
+import { ISelectableOption } from '../../utilities/selectableOption/SelectableOption.Props';
+
+export interface ISelectableDroppableTextProps<T> extends React.Props<T> {
+  /**
+  * Optional callback to access the ISelectableDroppableText interface. Use this instead of ref for accessing
+  * the public methods and properties of the component.
+  */
+  componentRef?: (component: T) => void;
+
+  /**
+   * Descriptive label for the ISelectableDroppableText
+   */
+  label?: string;
+
+  /**
+  * Aria Label for the ISelectableDroppableText for screen reader users.
+  */
+  ariaLabel?: string;
+
+  /**
+  * Id of the ISelectableDroppableText
+  */
+  id?: string;
+
+  /**
+   * If provided, additional class name to provide on the root element.
+   */
+  className?: string;
+
+  /**
+   * The key that will be initially used to set a selected item.
+   */
+  defaultSelectedKey?: string | number;
+
+  /**
+   * The key of the selected item. If you provide this, you must maintain selection
+   * state by observing onChange events and passing a new value in when changed.
+   */
+  selectedKey?: string | number;
+
+  /**
+   * Collection of options for this ISelectableDroppableText
+   */
+  options?: any;

```

Comment:
To allow for the options to be generic enough so that different implementers can utilize this as they see fit